submission/loglist_refresher_test.go

```diff
@@ -50,7 +51,8 @@ func ExampleLogListRefresher() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	f, err := createTempFile(`{"operators": [{"id":0,"name":"Google"}]}`)
+	data := bytes.NewBufferString(`{"operators": [{"id":0,"name":"Google"}]}`)
```

Comment:
Why not just pass this string directly to `createTempFile()`?

Reply:
Passed.